src/plugins/data/common/search/es_search/types.ts

```diff
@@ -21,11 +21,17 @@ import { IKibanaSearchRequest, IKibanaSearchResponse } from '../types';
 
 export const ES_SEARCH_STRATEGY = 'es';
 
+export type ISearchRequestParams = {
+  body?: any;
+  index?: string;
```

Comment:
These lines aren't needed since they're included in `SearchParams` (which inherits from `GenericParams`).